### DISCUSSION TITLE
关于状态管理，可以使用 MobX-State-Tree 或其他状态管理库吗？

### DISCUSSION THREAD

**Discussion by HaveF** [NetEase/tango#177] [General]
首先，感谢网易开源以及你们的工作！这个看上去是个非常有前景的项目。

我有个问题是关于 [tango-boot](https://github.com/NetEase/tango-boot) 的。

这个库看上去蛮不错，挺像 [MobX-State-Tree](https://mobx-state-tree.js.org/intro/welcome)以及 [mobx-react-lite](https://github.com/mobxjs/mobx/tree/main/packages/mobx-react-lite)，少了一些功能，但又加进去了一些功能。

老实讲，我稍微有点担心这个库有没有什么坑（想等等，抱歉，没有贬义，只是单纯说成熟度），所以想问下，有没有使用 MobX-State-Tree 或其他状态管理库的思路？

另外，[创建页面 | 基于源码的低代码引擎](https://netease.github.io/tango-site/docs/boot/page)中所说的
> defineView 的实现依赖了 [react-easy-state](https://github.com/RisingStack/react-easy-state#creating-reactive-views)，感兴趣的同学可以访问对应的链接了解更详细的内容。

点开看了下 react-easy-state，是四年前更新的，真的是有点吓人....

**Comment by wwsun** [comment]
react-easy-state 本身很稳定性，实现也很轻量，底层核心是借助 @nx-js/observer-util 实现的。

前端状态管理库这块社区发展的实在太快了，但万变不离其宗，核心的实现思路其实并不复杂，尤其是在应用形态相对不变的情况下。

**Reply by HaveF** [reply]
谢谢
